_includes/moto.md

```diff
@@ -0,0 +1,82 @@
+## MOTO
+
```

Comment:
Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6182398)